github/misc.go

```diff
@@ -107,19 +107,19 @@ func (c *Client) ListCodesOfConduct(ctx context.Context) ([]*Conduct, *Response,
 	// TODO: remove custom Accept header when this API fully launches.
 	req.Header.Set("Accept", mediaTypeCodesOfConductPreview)
 
-	var conducts []*Conduct
-	resp, err := c.Do(ctx, req, &conducts)
+	var codeOfConducts []*CodeOfConduct
+	resp, err := c.Do(ctx, req, &codeOfConducts)
```

Comment:
Minor, but the plural is on the wrong word here. The method and API is called code**s** of conduct, yet this variable is code of conduct**s**.

You can make it much shorter, since it's being used in a very local context. Either `cs`, `codes`, or something like that.

Reference: https://github.com/golang/go/wiki/CodeReviewComments#variable-names.